db/migrate/20150813073829_remove_duplicate_detailed_guide_artefacts.rb

```diff
@@ -0,0 +1,29 @@
+class RemoveDuplicateDetailedGuideArtefacts < Mongoid::Migration
+  IGNORED_DIFF_ATTRIBUTES = %w[_id actions updated_at created_at slug business_proposition paths tags]
+
+  def self.up
+    Artefact.where(kind: "detailed_guide", slug: %r{^(?!guidance/)}).each do |unmigrated|
+      duplicate = Artefact.where(kind: "detailed_guide", slug: "guidance/#{unmigrated.slug}").first
+      if duplicate
+        diff = duplicate.attributes.except(*IGNORED_DIFF_ATTRIBUTES).diff(
+          unmigrated.attributes.except(*IGNORED_DIFF_ATTRIBUTES))
+
+        if diff.any?
+          puts "Merging #{unmigrated.slug}:"
+          p diff
+
+          unmigrated.attributes = diff
+          unmigrated.save!
+        end
+        unmigrated.actions += duplicate.actions.map { |a| a.action_type = "update"; a }
+
+        duplicate.destroy
+      end
+      unmigrated.set(:slug, "guidance/#{unmigrated.slug}")
```

Comment:
Do you not also need to update the `paths` field so that the correct routes will get registered?